scripts/post-release-notes-preview.py

```diff
@@ -15,26 +15,41 @@
 GIT_OWNER_AND_REPO = os.environ['GITHUB_REPOSITORY']
 
 def main():
-    # last version:
-    lastVersion = subprocess.check_output(['git', 'describe', '--abbrev=0', '--tags', 'origin/staging'], text=True, stderr=subprocess.STDOUT).strip()
+    pullRequestNumber = eventData['number']
```

Comment:
1. there are github packages for python that might do this code cleaner, just in case you want to use one of them :) 
2. do we want to add a try-except here?

Reply:
1. thank you :)
2. no, if GitHub passes me a pull request event that has no number for the pull request, they have either changed or broken their API, in which case I want to fail miserably